editor/editor_properties_array_dict.cpp

```diff
@@ -517,6 +543,62 @@ void EditorPropertyArray::drop_data_fw(const Point2 &p_point, const Variant &p_d
 		emit_changed(get_edited_property(), array, "", false);
 		update_property();
 	}
+
+	if (drag_data.has("type") && String(drag_data["type"]) == "nodes") {
+		Array nodes = drag_data["nodes"];
+		Array nodes_paths = Array();
+		Node *base_node = get_base_node();
+		nodes_paths.resize(nodes.size());
+		for (size_t i = 0; i < nodes.size(); i++) {
+			Node *node = get_tree()->get_edited_scene_root()->get_node(nodes[i]);
+			NodePath path = node->get_path();
+
+			if (!base_node && Object::cast_to<RefCounted>(get_edited_object())) {
```

Comment:
This is from `EditorPropertyNodePath::_node_selected()` in `editor_properties.cpp`. Not sure if that if statement is needed, I never reached this part of the code but I don't really know its purpose.